### PR TITLE
Yci/migrate core version endpoint

### PR DESCRIPTION
**Pull request by younesschrifi**
migration of core/version endpoint

### CODE REVIEW

**Review by flomonster — APPROVED** [review]
LGTM

File: editoast/src/views/mod.rs [hunk 1]
```diff
@@ -240,6 +255,25 @@ mod tests {
...
+    async fn core_version() {
+        let mut core = MockingClient::new();
+        core.stub("/version")
+            .method(reqwest::Method::POST)
+            .response(StatusCode::OK)
+            .body(format!(
+                "{{
+                \"git_describe\": null
+        }}"
+            ))
```
Comment (flomonster) [inline editoast/src/views/mod.rs]:
```suggestion
            .body(r#"{{"git_describe": null}}"# )
```

**Review by flomonster — APPROVED** [review]

**Review by Uriel-Sautron — APPROVED** [review]
Read front only. LGTM !!!

File: editoast/src/views/mod.rs [hunk 2]
```diff
@@ -127,19 +132,29 @@ async fn version() -> Json<Version> {
     })
 }
 
+#[get("/version/core")]
+async fn core_version(core: Data<CoreClient>) -> Json<Version> {
+    let mut response = CoreVersionRequest {}.fetch(&core).await.unwrap();
+    if response.git_describe.is_some() {
+        response.git_describe = None;
```
Comment (anisometropie) [inline editoast/src/views/mod.rs]: What’s the point to always return `None` ?

Comment (younesschrifi) [reply editoast/src/views/mod.rs]: In local mode, you are always obliged to return `None`. Locally, we don't set the OSRD_GIT_DESCRIBE envvar!

**Review by anisometropie — APPROVED** [review]

**Pull request merged**